addins/Cake.Tin.yml

```diff
@@ -0,0 +1,12 @@
+Name: Cake.Tin
```

Comment:
Should be filtered as it is not an addin

Reply:
Done/ Cake.Tin added to blacklist